zebra-chain/src/notes/memo.rs

```diff
@@ -10,7 +10,7 @@ use std::{cmp, convert::TryFrom, fmt};
 ///
 /// [ps]: https://zips.z.cash/protocol/protocol.pdf#notept
 #[derive(Clone)]
-pub struct Memo(Box<[u8; 512]>);
+pub struct Memo(pub(crate) Box<[u8; 512]>);
```

Comment:
Wow, that is a *lot* of data per transaction.

Reply:
Weeeeeee!

Reply:
Yeah, if we were doing it again we'd probably have one memo per transaction, not per shielded output.